selectors/general.ts

```diff
@@ -29,6 +37,7 @@ export function getBasePath(state) {
 }
 
 export const getCurrentUserTimezone = createSelector(
+    'getCurrentUserTimezone',
```

Comment:
I'm a bit worried that this didn't break before now. This seems like this should've broken months ago when Joram added the custom version of `createSelector`. At least it's fixed now though